model/ExpandedLicensing/Individuals/NoAssertionLicense.md

```diff
@@ -3,19 +3,21 @@ SPDX-License-Identifier: Community-Spec-1.0
 # NoAssertionLicense
 
 ## Summary
+An Individual Value for License when no assertion can be made about it's actual value.
 
-Individual representing an absence of an assertion about license information.
 
 ## Description
 
-A NoAssertionLicense is the value that is used to indicate that the data creator
-is making no assertion about the license information for the corresponding software artifact.
+NoAssertion should be used if the SPDX creator has attempted to but cannot reach a reasonable objective determination;
```

Comment:
For the first word, "NoAssertion" or "NoAssertionLicense" or "NOASSERTION"?

I'd think "NoAssertionLicense" if it's referring to the Individual's title, or "NOASSERTION" if it's referring to the concrete string value.

Reply:
Updated to use NoAssertionLicense